Fix return value for RTCPeerConnection.addTransceiver() (#19871)
mdn/content: files/en-us/web/api/rtcpeerconnection/addtransceiver/index.md

### Return value

> The {{domxref("RTCRtpTransceiver")}} object which will be used to transmit and/or receive the media data.

Reviewer: ```suggestion
The {{domxref("RTCRtpTransceiver")}} object which will be used to exchange the media data.
```
